examples/notebooks/02_model_comparison.ipynb

```diff
@@ -165,9 +188,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 7,
    "metadata": {},
-   "outputs": [],
+   "outputs": [
```

Comment:
You need to clean your notebook.

I would suggest doing:
`git checkout 00876b7 -- examples/notebooks/02_model_comparison.ipynb`

Then remake your edits without running the notebook, and then commit.